daemon/network.go

```diff
@@ -3,14 +3,14 @@ package daemon
 import (
 	"fmt"
 	"net"
-	"sort"
 	"strings"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/api/errors"
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/network"
 	clustertypes "github.com/docker/docker/daemon/cluster/provider"
+	"github.com/docker/docker/pkg/plugins"
```

Comment:
Got it. thanks @anusha-ragunathan . That also answers @mrjana's question on the expectations.